lib/sync_checker/formats/edition_base.rb

```diff
@@ -66,7 +66,8 @@ def checks_for_draft(locale)
               expected_details_hash(edition_expected_in_draft)
             end
           ),
-          Checks::TranslationsCheck.new(edition_expected_in_draft.available_locales)
+          Checks::TranslationsCheck.new(edition_expected_in_draft.available_locales),
+          Checks::HttpStatusCheck.new(edition_expected_in_draft)
```

Comment:
Perhaps the route of LinksCheck is the way forward - we could pass in a range of http codes that we want to fail on - is that what you meant by https://github.com/alphagov/whitehall/pull/3043#commitcomment-20910818?